.travis.yml

```diff
@@ -23,13 +24,11 @@ matrix:
       env:
         - ARCH=arm
         - OS_VERSION=jessie
-        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
     - compiler: "gcc"
       dist: xenial
       env:
         - ARCH=arm
         - OS_VERSION=stretch
-        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
     - compiler: "gcc"
       os: linux
       env:
```

Comment:
EXTRA_SSH will be set for this configuration. And the next one (with clang). Before these 2 configurations did not have EXTRA_SSH set to anything, now it will be set. Just wondering if this might cause a problem.

Reply:
it won't [as discussed offline]